src/AWS.Logger.Core/Core/AWSLoggerCore.cs

```diff
@@ -636,11 +616,10 @@ public void AddMessage(InputLogEvent ev)
                 _request.LogEvents.Add(ev);
             }
 
-            public void Reset(string SeqToken)
+            public void Reset()
```

Comment:
I believe this is internal to Core so safe to adjust the signature, but let me know if you foresee any backwards compatibility issues.

Reply:
Yeah, I'm fine with this change to a public method because users shouldn't be directly interacting with this type.